connectors/src/api/create_connector.ts

```diff
@@ -169,11 +173,18 @@ const _createConnectorAPIHandler = async (
     return res.status(200).json(connector.toJSON());
   } catch (e) {
     logger.error(errorFromAny(e), "Error in createConnectorAPIHandler");
+    let errorMessage = `An unexpected error occured while creating the ${req.params.connector_provider} connector`;
+    const maybeInnerErrorMessage = (e as Error).message;
```

Comment:
Fair, although if for whatever reason what is thrown is not an `instanceof Error` but some other object with a `message` prop, it might still be desirable to obtain the `message` no ?

Reply:
Then we can probably have a typeguard that covers both?

Reply:
just to be sure I understand, what's the downside of casting to try and optain a potential message fro the unknown object ?